index.js

```diff
@@ -150,42 +157,37 @@ class SafeView extends Component {
     const WIDTH = isLandscape ? X_HEIGHT : X_WIDTH;
     const HEIGHT = isLandscape ? X_WIDTH : X_HEIGHT;
 
-    this.view._component.measureInWindow((winX, winY, winWidth, winHeight) => {
-      if (!this.view) {
-        return;
-      }
-      let realY = winY;
-      let realX = winX;
+    let realY = layout.y;
```

Comment:
this will give the x and y relative to the container, not the x and y relative to the entire *window*.  we need to know the x/y relative to the window because that's the only way to know if the container overlaps with a safe area. again run https://snack.expo.io/@notbrent/thrilled-pizza and notice the logs for "nested" show x/y of 0,0 despite it not being in the top left of the screen.